### PR TITLE
Add VDS13/telegram-inline-calendar

### PR DESCRIPTION
**Pull request by VDS13**

### CODE REVIEW

**Comment by VDS13** [comment]
Merged [PR](https://github.com/VDS13/telegram-inline-calendar/pull/3) and [fixed examples](https://github.com/VDS13/telegram-inline-calendar/commit/e1444c67cbfd169c1f13742c7b8c138f91a23117) in `EXAMPLES.md`.

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 - [Web Apps](#web-apps)
 - [Templates](#templates)
 - [Learning Resources](#learning-resources)
+- [Libraries](#libraries)
```
Comment (roj1512) [inline README.md:13]:
```suggestion
- [Plugins](#plugins)
```

File: README.md [hunk 2]
```diff
@@ -58,3 +59,7 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 ## Learning Resources
 
 - [Desarrollo de bots de Telegram en TypeScript](https://www.youtube.com/playlist?list=PL7q-McYJyHlip2-k4RW1nDxJWsFuO4nO1) - A free course in Spanish on creating bots using grammY.
+
+## Libraries
```
Comment (roj1512) [inline README.md:63]:
```suggestion
## Plugins
```

File: README.md [hunk 3]
```diff
@@ -58,3 +59,7 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
...
+
+- [VDS13/telegram-inline-calendar](https://github.com/VDS13/telegram-inline-calendar) - Date and time picker and Inline calendar for Node.js telegram bots.
```
Comment (roj1512) [inline README.md:65]:
```suggestion
- [VDS13/telegram-inline-calendar](https://github.com/VDS13/telegram-inline-calendar) - Date picker and inline calendar for your bots.
```

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much!

**Pull request merged**